packages/lib-classifier/src/store/markings/LineStore.js

```diff
@@ -0,0 +1,97 @@
+import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
+import { autorun } from 'mobx'
+import { filter, skip } from 'rxjs/operators'
+
+import Mark from './Mark'
+
+const Line = types
+  .model('LineStore', {
```

Comment:
The naming of these models confuses me. Does this model represent an individual line, or a store to hold lines?

Reply:
In that case, I'd suggest following the other model names (eg. `Classification` vs. `ClassificationStore`) and name this `Line` or `LineMark`.